### ISSUE TITLE
RoboMongo 0.9.0 : Script executed successfully, but there is no results to show

### ISSUE THREAD

**Issue by tishakariya** [paralect/robomongo#1269]
while executing command
   **db.getCollection('collectioname').find({});**
Output shown is:
   **Script executed successfully, but there is no results to show**
Where as there are (3000) documents in the collection, is there any issue??
RoboMongo Version : 0.9.0 Final
OS: Windows 7

**Comment by juliashibalko** [comment]
Hi @tishakariya, thanks for reporting the issue!
Could you please try this solution, it solved a lot of problems in the past: https://github.com/paralect/robomongo/issues/1106#issuecomment-230258348

Please let us know how it went.

**Comment by tishakariya** [comment]
Hey, Thanks for your reply @juliashibalko, it did not work with that,
still the same error.
The server i am trying to access is a Cloud server running on MongoDB 3.4.0.

**Comment by juliashibalko** [comment]
@tishakariya thanks for details. Could you please share more details to help us with reproducing:
1. Are you using connection with SSL, SSH or Authentication (if yes, is this admin)?
2. If it possible, could you tell us more about documents stored in DB? Do they have big size?
Thanks in advance!

**Comment by timkarnold** [comment]
Seeing this randomly as well. Seems influenced network latency, as it was occurring more frequently when tethered with a slow data connection. Re-attempting the query usually return results after one or more additional attempts.

1) Authentication as admin, no SSL
2) Seems to happen on both large and small size document queries

**Comment by Getz85** [comment]
Same issue here. It looks like the message is displayed only on time-consuming request (ie > 5 seconds).

For information, I tried with all 0.9.0-RCX version since the RC6, and the problem appeared with the 0.9.0-RC8 release.

So I suggest to other user with that problem to switch temporarily to RC7 version.

**Comment by juliashibalko** [comment]
Hello guys! 
@tishakariya could you please check issue with Robo3T v.1.1 and share results if possible? We implemented support of 3.4 MongoDB and I hope the issue won't be reproduced under this version. Please see new release here https://robomongo.org/download.

@timkarnold @Getz85 could you please try this solution, it solved a lot of problems in the past (increase shell timeout in `Options > Change Shell Timeout`: https://github.com/Studio3T/robomongo/issues/1106#issuecomment-310334178. Check blog http://blog.robomongo.org/robomongo-is-robo-3t/#4a. Please let us know how it went.

**Comment by andreeaflazar** [comment]
<img width="851" alt="Schermata 2020-11-10 alle 10 24 59" src="https://user-images.githubusercontent.com/65392192/98654790-0c3a6f00-233f-11eb-9389-e73caf25b576.png">
I have the same problem here, my classmates have a result but I get this message. I'm using Mongo 4.1.4 and the latest version of Mongodb. I don't know how to fix it...

**Comment by Panneer604306** [comment]
> ![robo](https://cloud.githubusercontent.com/assets/7137436/21601114/3feb1d6a-d1ac-11e6-89e3-2beddd2fae52.png) while executing command **db.getCollection('collectioname').find({});** Output shown is: **Script executed successfully, but there is no results to show** Where as there are (3000) documents in the collection, is there any issue?? RoboMongo Version : 0.9.0 Final OS: Windows 7

yes you can change the command while inserting data choose insert instead of insertMany
